Assets/Scenes/EndlessLevel.unity

```diff
@@ -11320,20 +11336,8 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   inventoryButtonTemplate: {fileID: 746893025}
-  imagesList:
-  - {fileID: 21300000, guid: a1d41b9f6cac6e84bbf052a0be48c710, type: 3}
-  - {fileID: 21300000, guid: a04f14fc5ea548247abaf7ba9c325aac, type: 3}
-  - {fileID: 21300000, guid: 34e6a9796bb0d1948904056eaa3ef18a, type: 3}
-  - {fileID: 21300000, guid: 398272e43eb962640bf4bb06f2c7dc3c, type: 3}
-  - {fileID: 21300000, guid: a04b0a1ce478f5a41ada67a58345a990, type: 3}
-  - {fileID: 21300000, guid: f7b667ec29a061e489152f558ca4cfc2, type: 3}
-  productNameList:
-  - Potion
-  - Superpotion
-  - Hyperpotion
-  - Phaser charge
-  - Laser charge
-  - Cannon charge
+  imagesList: []
```

Comment:
Could you recreate the imagesList and productNameList in this gameobject? Otherwise, inventory Will not be able to load icons in the pause menu.
If not, I can do It in a moment

Reply:
Oops somehow they disappeared by branch merging. I think it will take me much longer to figure out what should be where, would be awesome if you bring them back. 👍🏻